### PR TITLE
fix(TableContainer): ensure bottom border on all tables

### PR DESCRIPTION
**Pull request by tujoworker**
This PR fixes an issue introduced in #1956 where the first table in a container, has no bottom border.

Screenshot 2023-02-02 at 11 39 54

### CODE REVIEW

File: packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap [hunk 1]
```diff
@@ -56,15 +56,18 @@ exports[`Table scss have to match default theme snapshot 1`] = `
...
 > .dnb-table__tr.dnb-table__tr--has-accordion-content:not(.dnb-table__tr--expanded):nth-last-child(2) td::after {
     content: '';
     position: absolute;
+    top: 0;
     left: 0;
     right: 0;
     bottom: 0;
-    border-bottom: var(--outline); }
+    z-index: 1;
+    pointer-events: none;
```
Comment (NancyBolstad) [inline packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap:69]: Maybe it's possible to use `@include tableBorder()` mixins?

**Review by NancyBolstad — APPROVED** [review]
LGTM 👍

File: packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap [hunk 2]
```diff
@@ -665,10 +665,22 @@ thead > tr > th.dnb-table--active .dnb-table__sort-button.dnb-button:hover:focus
...
+      padding-top: 1.5rem; }
+    .dnb-table__container__body .dnb-table::after {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      z-index: 1;
+      pointer-events: none;
```
Comment (NancyBolstad) [inline packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap:682]: Maybe `@include tableBorder()`?

Comment (tujoworker) [reply packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap:682]: We do it there. This file is the output from SCSS when its rendered. So its pure CSS and is used to perform snapshot comparison.

File: packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/table/__tests__/__snapshots__/Table.test.tsx.snap:69]: We do it there. This file is the output from SCSS when its rendered. So its pure CSS and is used to perform snapshot comparison.

**Pull request merged**